lib/web/fetch/index.js

```diff
@@ -120,8 +122,49 @@ class Fetch extends EE {
   }
 }
 
+// publishIfSubscribed will check if there is context if so,
+// it will add error in the case of an error before publishing
+// the event if not it will return right away
+function publishIfSubscribed (name, context, error) {
```

Comment:
now this function does not check if it is subscribed, so the name is now wrong?

Reply:
Sure I can change it.